core/ir/opnd_api.h

```diff
@@ -1770,7 +1775,10 @@ struct _opnd_t {
          * above, to save space.
          */
         instr_t *instr; /* INSTR_kind, FAR_INSTR_kind, and MEM_INSTR_kind */
-        reg_id_t reg;   /* REG_kind */
+        struct {
+            reg_id_t reg;
+            opnd_size_t element_size;
+        } reg_and_element_size; /* REG_kind */
```

Comment:
We do expose this struct for DR_FAST_IR and while clients should be using the inlined accessor routines for source compatibility and the "reg" field should be at the same offset for binary compatibility it still seems worth mentioning in the release notes (as a non-compatibility-affecting change?  or a minor source compatibility change if someone were directly accessing the "reg" field??)

Reply:
Done